pkgs/development/python-modules/tinyarray/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  numpy,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "tinyarray";
+  version = "1.2.4";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-7NNCj9iki2H8XwpBPt4D4n2zod1T/NSeJKNtEaiimro=";
+  };
+
+  nativeBuildInputs = [
+    numpy
+    setuptools
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ pname ];
```

```suggestion
  pythonImportsCheck = [ "tinyarray" ];
```
Misuse of pname here will break this package if pname is overridden.